build-multiarch/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM golang:1.13 as builder
+ADD . /opt/kubesphere/
+WORKDIR /opt/kubesphere/
+RUN make all
+
+
+FROM ubuntu:18.04 as binarydownloader
+ARG HELM_VERSION=v3.5.2
+ARG KUSTOMIZE_VERSION=v4.0.5
+RUN apt update && apt -y install wget
+ADD ./build-multiarch/build-scripts /opt/build-scripts/
```

Comment:
How about doing all the work in one container? So we don't need to pull two images.

Reply:
I don't think so. The advantage of multi-stage construction is that we can do things in different environments. Consider this, if we want to upgrade the version of golang, we just need to change the tag of builder and do not need do anything with `binarydownloader`.